src/git.rs

```diff
@@ -210,24 +206,6 @@ impl Git for LiveGit {
         Ok(commits)
     }
 
-    fn find_unpushed_commits(
```

Comment:
This was unused?
Perhaps we should add a tool to check for unused functions?

Reply:
Generally speaking yes, we do already have this. It doesn't work in this case because this trait is part of this crate's public interface. It has to be part of the public interface because the actual implementation of the trait is only passed into the functions that use it from the binary crate.

Technically, this is a breaking change on the interface of this library crate, that I can get away with because it isn't published or used outside this repo.